qiskit_sphinx_theme/__init__.py

```diff
@@ -27,10 +27,20 @@ def get_html_theme_path():
         stacklevel=2,
         category=DeprecationWarning,
     )
-    return _get_theme_absolute_path("pytorch_base")
+    return _get_theme_absolute_path("pytorch/ecosystem")
 
 
 # See https://www.sphinx-doc.org/en/master/development/theming.html
 def setup(app):
-    app.add_html_theme("qiskit_sphinx_theme", _get_theme_absolute_path("pytorch_base"))
+    # Note: base themes should not be exposed in the entry_points for the package
+    # (in setup.py/pyproject.toml). We only need to register the theme for `inherit` in `theme.conf`
+    # to work.
+    app.add_html_theme("__qiskit_pytorch_base", _get_theme_absolute_path("pytorch/base"))
+
+    app.add_html_theme('qiskit_legacy', _get_theme_absolute_path("pytorch/terra"))
```

Comment:
It feels a little weird to have a naming mismatch between the theme `qiskit_legacy` and the directory `*/terra`, but it's an internal detail, so if it's easier for you guys to work with then no problem.